.github/workflows/golangci-lint.yaml

```diff
@@ -0,0 +1,36 @@
+name: golangci-lint
+
+on:
+  pull_request:
```

Comment:
How about adding `paths`. We can run the go lint only when PR involves changes in golang file. Suppose PR involves changes only in the README file, then we don't want the lint to run.  (https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths)

Reply:
Done